r/adbcsnowflake/DESCRIPTION

```diff
@@ -0,0 +1,26 @@
+Package: adbcsnowflake
+Title: Arrow Database Connectivity (ADBC) Snowflake Driver
+Version: 0.0.0.9000
+Authors@R: c(
+      person("Dewey", "Dunnington", , "dewey@dunnington.ca", role = c("aut", "cre"),
+             comment = c(ORCID = "0000-0002-9415-4582")),
+      person("Apache Arrow", email = "dev@arrow.apache.org", role = c("aut", "cph"))
+    )
+Description: Provides a developer-facing interface to the Arrow Database
+  Connectivity (ADBC) Snowflake driver.
+License: Apache License (>= 2)
+Encoding: UTF-8
+Roxygen: list(markdown = TRUE)
+RoxygenNote: 7.2.3
+SystemRequirements: golang
```

Comment:
Is this still a requirement now if you've set it up to download Go locally and install if needed?